docs/source/models/risk_correlation/sbp_ldlc_fpg_bmi/index.rst

```diff
@@ -0,0 +1,89 @@
+.. _2023_sbp_ldlc_fpg_bmi:
+
+-------------------------------------------------------
+SBP, LDL-C, FPG, and BMI Risk Factor Correlation - 2023
+-------------------------------------------------------
+
+Risk Exposure Overview
+----------------------
+
+SBP, LDL-C, FPG, and BMI are all continuous risk exposures that affect 
+different cardiovascular disease outcomes. In the 2023 model for 
+CVD, these outcomes are heart failure incidence, myocardial infarction, and 
+ischemic stroke. 
+
+It is also known that these 4 risks are correlated in the population. To properly 
+account for the uneven distribution of risk, we are including a correlation between 
+the relevant risk factors (more detail on which are included below). 
+
+Risk Exposures in GBD 
+----------------------
+
+Links to documentation for relevant risk exposure pages include:
+
+- :ref:`SBP risk exposure <2019_risk_sbp>`
+
+- :ref:`LDL-C risk exposure <2019_risk_exposure_ldl>`
+
+- :ref:`FPG risk exposure <2019_risk_exposure_fpg>`
+
+- :ref:`BMI risk exposure <2019_risk_bmi>`
+
+Vivarium Modeling Strategy
+----------------------------
```

Comment:
Maybe this is coming after, but I think we might need some more detail on _how_ we intend to enforce these correlations. Mainly...

- If we enforce correlation between risk A and B as well as B and C, then there's no guarantee that the resulting correlation between A and C will match what we got out of NHANES. We might need to make a ranked priority between risk pairs to get ahead of this (hopefully it all matches up given that we're using the same data source for everything, but we'll see!).
- I assume that the correlation should be applied to the *untreated* exposure values for these risks and then treatment effects that are present at initialization will be layered on top. Is that right? We should make that clear if so!

Reply:
Addressing your first bullet point - maybe I am not understanding something, but I thought we were enforcing A to B, B to C AND A to C. The multivariate normal function has correlations for SBP to BMI and BMI to LDL-C but also SBP to LDL-C. Code is below - is this not doing what I think it is doing? 

`probit_propensity = np.random.multivariate_normal([0,0,0,0], [[1, sbp_ldl, sbp_bmi, sbp_fpg],
                                                         [sbp_ldl, 1, bmi_ldl, ldl_fpg],
                                                         [sbp_bmi, bmi_ldl, 1, bmi_fpg],
                                                             [sbp_fpg, ldl_fpg, bmi_fpg, 1]], size=N)`

Reply:
Oh yeah sorry, just took a look and this is a smarter function than I thought. Yeah disregard my initial point then!